tools/devtools/utils.mjs

```diff
@@ -0,0 +1,280 @@
+  let enContent =
+    enFile.match(
+      /(?<=<!-- \$contentStart -->)(.+?)(?=<!-- \$contentEnd -->)/s
+    )?.[0] || '';
+
+  const enParagraphs = (
+    enContent.match(
+      /(?<=\n)((?!{%|{#|Chromium issue: |Chromium issues: ).*)(?=\n)$/gm
+    ) || []
+  ).filter(x => x);
+
+  for (const p of enParagraphs) {
+    if (p.startsWith('```'))
+      console.warn(
+        "Output contains code sample. Please check if it's commented correctly."
+      );
+
+    enContent = enContent.replace(p, `<!-- ${p} -->`);
+  }
+
+  // Append the English description and commented content to translation files
+  for (const lang of langs) {
+    const fileName = nunjucks.renderString(blogDest, {lang, version});
+
+    const output = nunjucks.render(fileName, {
+      desc: enDesc,
+      content: enContent,
+    });
+
+    await writeFile(fileName, output, 'utf-8');
+  }
+  console.log(`Populated commented content for: ${langs.join(', ')}`);
+}
+
+/**
+ * Append What's New in DevTools outline in 7 languages
+ * @param {String} version - release version
+ * @param {String[]} langs - languages you want to generate
+ */
+export async function createWndtOutline(version, langs) {
+  // Read the file
+  const fileName = nunjucks.renderString(blogDest, {lang: 'en', version});
+  const content = await readFile(fileName, 'utf-8');
+
+  // Abstract the title and hash id
+  const list = (content.match(/^(## |### )(.*)$/gm) || []).map(x => {
+    const title = x.match(/(?<=## )(.*?)(?= {:)/gm);
+    const hash = x.match(/(?<={: )(.*?)(?= })/g);
+
+    return `* [${title}]({{hashLang}}/blog/new-in-devtools-{{version}}/${hash})`;
+  });
+
+  const output =
+    '### Chrome {{version}} {: #chrome{{version}} }\n\n' +
+    list.join('\n') +
+    '\n';
+
+  // Append outline to translation files
+  for (const lang of langs) {
+    let langOutput = nunjucks.renderString(output, {
+      hashLang: lang === 'en' ? '' : `/${lang}`,
+      version,
+    });
+
+    if (lang !== 'en') {
+      langOutput = `{# ${langOutput} #}`;
+    }
+
+    const outlineFileName = nunjucks.renderString(outlineDest, {lang});
+    let outlineFileContent = await readFile(outlineFileName, 'utf-8');
+
+    const contentHolder = '{# $content #}';
+    outlineFileContent = outlineFileContent.replace(
+      contentHolder,
+      '{{ content }}'
+    );
+
+    const out = nunjucks.renderString(outlineFileContent, {
+      content: contentHolder + '\n\n' + langOutput,
+    });
+
+    await writeFile(outlineFileName, out, 'utf-8');
+  }
+
+  console.log(`Appended WNDT outlines for: ${langs.join(', ')}`);
+}
+
+/**
+ * Create Github Issues for translators to work on
+ * @param {String} version - release version
+ * @param {String} dueDate - due date to translate
+ * @param {String[]} langs - languages you want to generate
+ * @param {String} auth - GitHub Token
+ * @param {Object} translators - Translators of different locales { en: 'githubHandler' }
+ */
+export async function createGitHubIssues(
```

Comment:
I wonder if it's worth safe guarding this to prevent issues from getting created twice if the script is run a second time with the same variables by accident?

Reply:
I have a safe guard when calling this function. The CLI will explicitly ask if we want to generate, see `wndt-translation-bootstrap` - `readline.question('Do you want to generate GitHub issues? y/[n]:` 

The worst case is we have duplicated issues that we need to manually clean up.

Reply:
Yep, I have seen this one 🙂 I just can imagine myself seeing pressing `y` in a rush - that's where I was coming from. But maybe even calling out the configured version in the prompt again would save me already.

Reply:
Add version number and force it to be capital Y for additional safeguard.